.github/workflows/build.yml

```
@@ -53,3 +53,96 @@ jobs:
             -   name: Show errors
                 if: ${{ failure() }}
                 run: ./.build.scripts/show-errors.sh
+
+    windows:
+        runs-on: windows-latest
+        name: "Windows: Build and test"
+        defaults:
+            run:
+                shell: cmd
+        strategy:
+            matrix:
+                #php: ["7.2", "7.3", "7.4", "8.0", "8.1"]
+                php: ["7.3", "7.4", "8.0", "8.1"]
+                arch: [x64, x86]
+                #ts: [nts, ts]
+                ts: [ts]
+        steps:
+                - name: Checkout Xdebug
+                  uses: actions/checkout@v2
+                - name: Extract Xdebug Version
+                  shell: powershell
+                  run: |
+                    chcp 65001
+                    $r = Select-String -Path php_xdebug.h -Pattern 'XDEBUG_VERSION\s+"(.*)"'
+                    $s = $r.Matches[0].Groups[1]
+                    echo "$s"
+                    $xdebug_version = 'XDEBUG_VERSION=' + $s
+                    echo $xdebug_version >> $env:GITHUB_ENV
+                - name: Setup PHP
+                  id: setup-php
+                  uses: cmb69/setup-php-sdk@v0.1
```

```suggestion
                  uses: cmb69/setup-php-sdk@v0.2
```
Should fix the nts/ts issue. Was a [stupid bug](https://github.com/cmb69/setup-php-sdk/commit/dc2a4d88bb1cb3484e9b4a5c010cb8b90d751147). Thanks for reporting!